src/formatDate.js

```diff
@@ -50,7 +50,33 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
-}
+  const arrDate = date.split(`${fromFormat[3]}`);
+  let str = '';
+
+  const obj = {
```

Comment:
the same about variable name
and it could be just an empty object, because this `obj[fromFormat[i]] = arrDate[i];` will define new property even if it doesn't exist in your object before